### PR TITLE
Html forms

### PR DESCRIPTION
**Pull request by KennethDelaCruz**

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title> HTML Forms</title>
+  </head>
+
+  <body>
+    <form>
+      <p> Email
```
Comment (codingcodymiller) [inline html-forms/index.html]: This should be a `label` element

Comment (codingcodymiller) [reply html-forms/index.html]: Also, please remove the `p` element holding all the form content, it is unnecessary 👍

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+          <br />
+            <input type="text" name="Email" size="15"
+             max length="30"/>
+             <br />
+          Why is coding cool?
```
Comment (codingcodymiller) [inline html-forms/index.html]: This should be a `label` element

File: html-forms/index.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+              <br />
+            <select name="Why is coding cool">
+              <option value="Select an Option">Select a Value</option>
+              <option value="Because it's interesting">Because it's Interesting</option>
+              <option value="Because it's fun">Because it's fun</option>
+              <option value="Because it's powerful">Because it's powerful</option>
+            </select>
+            <br />
+          Favorite Language (choose one) <br />
+          <input type="radio" name="favorite language" value="HTML" /> HTML
```
Comment (codingcodymiller) [inline html-forms/index.html:24]: The text for these should be in a `label` element

File: html-forms/index.html [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+          <input type="radio" name="favorite language" value="CSS" /> CSS
+          <input type="radio" name="favorite language" value="JAVASCRIP" /> JAVASCRIPT
+          <br />
+          Would you like to sign up for more information?
+          <br />
+          <input type="checkbox" name="email signup" value="email" />Sign me up!
```
Comment (codingcodymiller) [inline html-forms/index.html:30]: Text for this should be in a `label` element

File: html-forms/index.html [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+          <br />
+          Comments?
```
Comment (codingcodymiller) [inline html-forms/index.html]: This should be a `label` element

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Please add a `for` attribute to each `label` element, with the correct value to tie it to its `input` element 👍

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Kenn! You still have a couple of comments that have not been addressed. Also, please attach a screenshot of your output with each exercise solution :+1:

**Comment by KennethDelaCruz** [comment]
image

File: html-forms/index.html (hunk 5 above)
Comment (KennethDelaCruz) [reply html-forms/index.html]: image

File: html-forms/index.html (hunk 4 above)
Comment (KennethDelaCruz) [reply html-forms/index.html:30]: image

File: html-forms/index.html (hunk 3 above)
Comment (KennethDelaCruz) [reply html-forms/index.html:24]: image

File: html-forms/index.html (hunk 1 above)
Comment (KennethDelaCruz) [reply html-forms/index.html]: image

**Comment by thebearingedge** [comment]
@KennethDelaCruz Hi Kenn, Please make another commit and push to this branch and open a new PR so we can leave a review.

**Pull request merged**